packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -177,6 +177,21 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = (
             borderWidth: "0",
             justifyContent: "flex-start",
         },
+        button__stealth: {
+            minWidth: "32px",
+            maxWidth: "unset",
+            padding: "0",

```

Comment:
We shouldn't need to change anything other than colors here. They layout should be the same as the primary, secondary, outline, and lightweight styles.

Reply:
I think its open for discussion. The most common scenario for stealth button is it only displaying an Icon. The idea behind creating this appearance was to not have to override the width, padding and other properties every time we wanted this type of button. If someone wants a stealth button that displays like the other buttons they can just as easily add those overrides but that is not a common scenario.